handbook/product/design/ways_of_working/index.md

```diff
@@ -0,0 +1,26 @@
+
+## Working as embedded designers
+
+We prefer to work as embedded designers whenever possible. In this way of working, a product designer is a member of a given cross-disciplinary team (like Batch changes, Search product, Extensibility, or Cloud SaaS). Each team has members from engineering, product management, and product design. This team collaborates closely day-by-day to move closer to achieving the team’s goals and long-term vision.
+
+Working as an embedded designer is the best way for us to collaborate end-to-end across our design process.
+When working as an embedded designer in a cross-disciplinary team, we join that team’s syncs, planning, retros, and other rituals, and also participate in defining and reaching the team’s OKRs.
+
+## Working as design consultants
+
+We often work as design consultants when the need arises. This might be because a team is loosely defined around a project but isn’t formally defined as a cross-disciplinary team, and may not have an embedded product designer. Or it might be because we don’t have enough designers on our team to support each cross-disciplinary team, but a team still needs design support.
+
+Working as a design consultant gives us a way to contribute end-to-end on efforts that benefit from a product design perspective. What this means is a bit different for every consulting effort. Sometimes it means checking in with the team from time to time to share feedback. Other times, it means facilitating regular timeboxed design sessions with team members to share and explore design problems synchronously.
+
+It’s important than when we’re working as design consultants, we’re setting and managing expectations around time and outcomes. A consulting effort inherently can’t be as extensive or in-depth as it might be as an embedded designer, but it will unblock product efforts and help us to move forward towards our goals. Early design support, even in a consulting role, will help us as a company save aggregate time and effort in the future by making better design decisions early and avoiding future design and conceptual debt.
+
+## Working as a product design team
+
+We each work as a member of the product design team. As a team, we’re responsible for the discipline of product design at Sourcegraph. This includes defining the team culture, rituals, expectations, tooling, and processes, as well as supporting each other’s growth and helping each other to be successful as product designers at Sourcegraph.
+
+As a product design team, we create space for our own [team rituals](../team_rituals/index.md), regularly request and provide feedback in [design critiques](../design_critiques/index.md), and advocate for our [design principles](../../product_management/index.md#sourcegraph-s-design-principles) through our actions.
```

Comment:
```suggestion
As a product design team, we create space for our own [team rituals](../team_rituals/index.md), regularly request and provide feedback in [design critiques](../design_critiques/index.md), advocate for our [design principles](../../product_management/index.md#sourcegraph-s-design-principles) through our actions, and promote design thinking and creativity across Sourcegraph.
```

Reply:
Just a suggestion :) I would personally like to take on more conscious efforts to promote creativity and innovation across the company as a design team.